graph/builder.go

```diff
@@ -1206,6 +1206,16 @@ func (b *Builder) processUpdate(msg interface{},
 				"chan_id=%v", msg.ChannelID)
 		}
 
+		// Look up the funding pk script so that we can register the
```

Comment:
> In case we assume valid channels or have alias channels we make
sure we also add the channel point to the block filter so that
we are notified if the channel point is spent in a block.

Did you encounter this or can you demonstrate the behavior in a test? I think we have itests for the alias channels and the UTXO filter seems to work fine?

Reply:
yes the zeroconf channel test is failing if we don't do this because we would never be notified if the channel closed because for alias channels we are not funneling through the channel_announcement again as before, we are just swapping it in the ReAssignmentSCID method, before we would send it again via `addToGraph`.